test/dynamodb/converter.spec.js

```diff
@@ -237,6 +237,19 @@ describe('AWS.DynamoDB.Converter', function() {
       );
     });
 
+    describe('sets', function() {
+      it(
+        'should be iterable',
+        function() {
+          var list = [];
+          for (const item of new DynamoDBSet([1, 2, 3])) {
```

Comment:
move `[1, 2, 3]` to a new list, say `inputList` for example.
The variable list can be similarly be renamed to `outputList`

Reply:
Updated